frontend/src/actionCreators/parcelsActionCreator.tsx

```diff
@@ -64,18 +65,34 @@ export const fetchPropertyDetail = (id: number, propertyTypeId: 0 | 1) => (dispa
     : dispatch(fetchBuildingDetail({ id }));
 };
 
-export const createParcel = (parcel: API.IParcel) => (dispatch: Function) => {
+export const createParcel = (parcel: IParcel) => (dispatch: Function) => {
   dispatch(request(actionTypes.ADD_PARCEL));
   dispatch(showLoading());
   return CustomAxios()
     .post(ENVIRONMENT.apiUrl + API.ADD_PARCEL, parcel, createRequestHeader())
     .then((response: AxiosResponse) => {
       dispatch(success(actionTypes.ADD_PARCEL, response.status));
-      dispatch(clear(actionTypes.ADD_PARCEL));
+      dispatch(fetchParcelDetail(response.data));
       dispatch(hideLoading());
     })
     .catch((axiosError: AxiosError) =>
       dispatch(error(actionTypes.ADD_PARCEL, axiosError?.response?.status, axiosError)),
     )
     .finally(() => dispatch(hideLoading()));
 };
+
+export const updateParcel = (parcel: IParcel) => (dispatch: Function) => {
```

Comment:
If I followed this code correctly we now have two, different, `IParcel` interfaces (one for create, one for update)...
Any chance you could try to find a way to rename one of them to avoid confusing? 
I wish I could provide a suggestion but I'm drawing a blank here...
> _There are only two hard things in Computer Science: cache invalidation and naming things._